sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/feast_tests.py

```diff
@@ -0,0 +1,79 @@
+# TODO: Do we want to keep these tests? Have not been re-validated with v0.19
```

Comment:
This file should be removed.
@adchia do you know how to add this plugin into the CI of Feast directly (and potentially making them optional to merge)